src/client/Client.cc

```diff
@@ -174,7 +174,7 @@ Client::Client(Messenger *m, MonClient *mc)
     tick_event(NULL),
     monclient(mc), messenger(m), whoami(m->get_myname().num()),
     cap_epoch_barrier(0),
-    last_tid(0), oldest_tid(0), last_flush_seq(0),
+    last_tid(0), oldest_tid(0), last_flush_tid(0),
```

Comment:
Since we're interpreting an oldest_flush_tid of 0 as meaning the client doesn't support this feature, and we don't want to leak the client's first flush, we should probably start last_flush_tid at 2 and the flushing_caps_tids filled in with a value of 1?